project/BuildDependencies/DownloadBuildDeps.bat

```diff
@@ -29,14 +37,26 @@ md %TMP_PATH%
 cd scripts
 
 SET FORMED_OK_FLAG=%TMP_PATH%\got-all-formed-packages
-REM Trick to preserve console title
+:: Trick to preserve console title
 start /b /wait cmd.exe /c get_formed.cmd
 IF NOT EXIST %FORMED_OK_FLAG% (
   ECHO ERROR: Not all formed packages are ready!
+  ECHO.
+  ECHO I tried to get the packages from %KODI_MIRROR%;
+  ECHO if this download mirror seems to be having problems, try choosing another from
+  ECHO the list on http://mirrors.xbmc.org/list.html, and setting %%KODI_MIRROR%% to
+  ECHO point to it, like so:
+  ECHO   C:\^> SET KODI_MIRROR=http://example.com/pub/xbmc/
```

Comment:
It's necessary. ^> is the escape sequence to print a literal ">". Without the ^, cmd.exe interprets the > as output redirection.